docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -540,6 +540,8 @@ Ansible ad hoc command is the easiest option:
 .. code-block:: shell-session
 
     ansible all -i localhost, -m debug -a "msg={{ 'mypassword' | password_hash('sha512', 'mysecretsalt') }}"
+    
+.. note:: Due to differences in the underlying tooling this method will, with identical input, generate different passwords on MacOS and Linux! 
```

Comment:
That is not entirely correct.  If passlib is installed you will get the same results.  Additionally the `crypt` method you are likely referencing on Linux has been deprecated.  Also the exclamation point at the end is rather unnecessary.

Reply:
Fair enough.